src/components/Calculator.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import calculate from './logic/calculate';
+
```

Comment:
- [ ] Inside calculator component should have been calculate and operate as imported. I see the operation is inside calculate, please, change that importation on `calculator.js`. This component should have both. 🐶